### ISSUE TITLE
Installation question

### ISSUE THREAD

**Issue by kbuzzard** [ImperialCollegeLondon/imperial_latex_templates#5]
tl;dr: what is the correct way to get my linux machine in a state so that I don't have to copy the `Fonts` and `Images` directories into many other places on my system?

I've just successfully used this repo to write a reference, by e.g. copying the `Fonts` and `Images` directories to subdirectories of the directory corresponding to the student (I have hundreds of subdirectories in my references directory, one per student). But this isn't a sustainable solution for me, and from the README my understanding is that I need to install this package directly. I'm on Ubuntu and my solution for the past decade or so has just been to install some kind of `texlive-full` package and never think any more about (and am essentially completely ignorant of) the behemoth that LaTeX has become (I am not doing anything fancy with LaTeX and this approach is in practice fine for me). Right now the installation instructions in this repo just refer me to https://en.wikibooks.org/wiki/LaTeX/Installing_Extra_Packages . 

So attempting to work through that page I'm slightly stymied by the fact that I no longer know anything about the LaTeX infrastructure. I started by reading the "Automatic installation" paragraph. Presumably I have a package manager installed: 

```bash 
$ which tlmgr
/usr/bin/tlmgr
```

but I am not clear about whether this repo is a "package" or not, and how to use this command to install it if it is. I tried

```bash
$ tlmgr install imperial_latex_templates/
(running on Debian, switching to user mode!)
(see /usr/share/doc/texlive-base/README.tlmgr-on-Debian.md)
TLPDB: not a directory, not loading: /home/buzzard/texmf
tlmgr: user mode not initialized, please read the documentation!
```

I'm slightly reluctant to run as root because I don't know what I'm doing.

So I move on to the "Manual installation" paragraph and the first thing it says is to look for the `.dtx` and `.ins` files, neither of which appear to be present. I conclude that I'm doing the wrong thing. Can I get some more hints? Many thanks! Right now I will stick with the workaround which is just multiple copies of the `Fonts` and `Images` directories.

**Comment by dham** [comment]
Manual installation is correct. This package doesn't have .dtx and .ins because the author didn't do it that way, but that just means that manual steps 1 and 2 don't apply so you continue from step 3. 

I agree that the documentation upstream could be clearer on this. It's a wiki so you can edit it if you think you can make it clearer.

**Comment by kbuzzard** [comment]
Steps 3 and 4 have gone fine, for Step 5 I am *assuming* that all the `.ttf` files in `Fonts/` are TrueType fonts (I don't know the first thing about fonts) but I can't find a `.map` file in the repo so I can't follow the instructions. Given that I haven't done Step 5 I am assuming that this is the cause of the
```
! Package fontspec Error: The font "ImperialSansText-Regular.ttf" cannot be
(fontspec)                found.
```
error I'm currently getting.
